ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/Permissions.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.hmcts.reform.divorce.ccd;
+
+public final class Permissions {
```

Comment:
I think this is in the latest version of the lib

Reply:
Looks like there are some changes that have not been released yet

Reply:
you can use https://github.com/hmcts/ccd-config-generator/releases/tag/0.15.2 now